src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -133,7 +133,8 @@ class_exists(__NAMESPACE__.'\TestingCaseMismatch', true);
      */
     public function testFileCaseMismatch()
     {
-        if (!file_exists(__DIR__.'/Fixtures/CaseMismatch.php')) {
+        //Cannot be tested if the file loads (there is is casemismatch.php in the filesystem)
+        if (file_exists(__DIR__.'/Fixtures/CaseMismatch.php')) {
```

Comment:
This test tests whether there is an exception thrown for non-existing file - see the `@expectedException` annotation (non-existing in this case means *with different case*).

It can't be tested on case insensitive filesystem (such as Windows) because the file is loaded anyway (even with different case). There is a `casemismatch.php` on [the filesystem](https://github.com/symfony/symfony/tree/2.8/src/Symfony/Component/Debug/Tests/Fixtures). So the check for `CaseMismatch.php` returns `false` on case sensitive filesystems (Linux) and the test is executed.

The problem is that the `class_exists` does not throw `RuntimeException` but issues an error.